SimTracker/TrackTriggerAssociation/interface/TTClusterAssociationMap.h

```diff
@@ -146,6 +163,12 @@ bool TTClusterAssociationMap< T >::isGenuine( edm::Ref< edmNew::DetSetVector< TT
   unsigned int goodDifferentTPs = 0;
   std::vector< const TrackingParticle* > tpAddressVector;
 
+  std::vector<float> tp_mom;
+
+  tp_mom.clear();
```

Comment:
no need to clear vector right after creating it...

Reply:
Done